rewrite-java/src/main/java/org/openrewrite/java/tree/J.java

```diff
@@ -757,6 +757,20 @@ public Block withStatements(List<JRightPadded<Statement>> statements) {
                 return t.statements == statements ? t : new Block(t.id, t.prefix, t.markers, t.statik, statements, t.end);
             }
         }
+
+        /**
+         * An empty {@link Block} with no contents.
+         */
+        public static J.Block empty() {
+            return new J.Block(
+                    Tree.randomId(),
+                    Space.EMPTY,
+                    Markers.EMPTY,
+                    JRightPadded.build(false),
+                    Collections.emptyList(),
+                    Space.EMPTY
+            );
+        }
```

Comment:
Can you move this into the `SimplifyConstantIfBranchExecutionVisitor`?

Reply:
I can, but why? This is a factory method for creating empty `J.Block` objects. I presume that my visitor will not be the only case where a visitor will want to return an empty `J.Block`.

Reply:
> I can, but why? 

To keep the domain and API as small and straightforward as is possible.  So far, there are more than 700 recipes and less than ten instances where `J.Block` is instantiated with an empty list of statements.